dev/com.ibm.ws.jsp/src/com/ibm/wsspi/jsp/taglib/config/TldPathConfig.java

```diff
@@ -69,9 +91,18 @@ public void setUri(String string) {
     
     /**
      * Gets whether the tld contains any listener elements
-     * return boolean - if the tld file contains any listener elements
+     * @return boolean - if the tld file contains any listener elements
      */
     public boolean containsListenerDefs() {
         return containsListenerDefs;
     }
+
+    /**
+     * Determines if the uri argument should override the uri attribute in the TLD
```

Comment:
Should we add how this is determined? Via calling one constructor over the other for now?